### DISCUSSION TITLE
Setting skins per ruleset

### DISCUSSION THREAD

**Discussion by dee-fox** [ppy/osu#28695] [Ideas] [answered]
Applying skins per ruleset doesn't seem useful (because, frankly, it isn't), but I just prefer some skins in some game modes, like the more readable classic skin in catch, or a custom skin where notes just seem a little bit clearer in mania. This feature request sounds dumb, mostly because it really is.

**Comment by peppy** [comment, marked as answer]
Definitely on our radar as a valid concern!

**Comment by Glebsin** [comment]
Setting skins for personal presets of mods would be a good idea too
